Packs/CommonScripts/ReleaseNotes/1_2_56.md

```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: IsInternalDomainName
+- Checks if a subdomain is located in the specified domain.
```

Comment:
@ShirleyDenkberg I think we should reconsider if this is a good phrasing, I don't think it accurately describes what the script does